content/03-type-system.md

```diff
@@ -221,7 +221,7 @@ Unification is the heart of the type system and contributes immensely to the rob
 
 > ##### Define: Unification
 >
-> Unification between two types A and B is a directional process which answers the question if A **can be assigned to** B. It may **mutate** either type if it is or has a [monomorph](types-monomorph).
+> Unification between two types A and B is a directional process which answers one question: if A **can be assigned to** B. It may **mutate** either type if it either is or has a [monomorph](types-monomorph).
```

Comment:
`… question: whether A can be …`

Reply:
I intentionally didn't change this out of fear I might mess up the original intent. Easy fix.